app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt

```diff
@@ -142,6 +141,54 @@ class SettingsFragment : Fragment(), SettingsView, AdapterView.OnItemClickListen
         }
     }
 
+    fun changeLanguage() {
+        val languages = resources.getStringArray(R.array.languages)
+        val mainActivity =(activity as? MainActivity)
+
+        context?.let {
+            AlertDialog.Builder(it)
+                .setTitle(R.string.title_choose_language)
+                .setSingleChoiceItems(languages, -1) { dialog, which ->
+                    when (which) {
```

Comment:
We support 15 languages on the app, you're supporting only 8 here. Any reason for that?

Reply:
Ok, will add other languages. I  have asked earlier about this [here ](https://github.com/RocketChat/Rocket.Chat.Android/pull/1918#discussion_r264835629

Reply:
Sorry for not replying you there. But don't expect the strings to be fully translated to support it on this feature.